src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/CommentsIndentation/context/defaultContextConfig.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="CommentsIndentation"/>
+
+        <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
+            <property name="file" value="${tp}/defaultContext.patch" />
+            <property name="strategy" value="context" />
+            <property name="neverSuppressedChecks" value="CommentsIndentationCheck" />
```

Comment:
Please add another subfolder but without this property to check that no violations are shown for the above case.

Reply:
done, no `neverSuppressedChecks` will have no violation.

Reply:
Please clarify why did you add `<property name="supportContextStrategyChecks" value="CommentsIndentationCheck" />`? Could you test this config and confirm there are no violations:
```
        <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
            <property name="file" value="${tp}/defaultContext.patch" />
            <property name="strategy" value="context" />
        </module>
```

Reply:
@rdiachenko replied here https://github.com/checkstyle/patch-filters/pull/168#discussion_r469755922